distage/distage-model/src/main/scala/izumi/distage/model/definition/dsl/ModuleDefDSL.scala

```diff
@@ -149,6 +153,25 @@ object ModuleDefDSL {
     }
   }
 
+  final class MultipleDSL[T]
+  (
+    protected val mutableState: MultipleRef
+  ) extends MultipleDSLMutBase[T] {
+
+    def to[I >: T : Tag]: MultipleDSL[T] = {
```

Comment:
Could you make it `class MultipleDSL[I]` & `[T >: I : Tag]: MultipleDSL[I]`? Because in this case the relationship between `I`mplementation and `T`ype is inverted...

Reply:
Done.